plugins/hdfsviewer/src/azkaban/viewer/hdfs/velocity/hdfs-proxy-js.vm

```diff
@@ -36,5 +37,11 @@ $(function() {
     };
     $.post("", requestData, successHandler);
   });
+
+  $('#proxyname').keyup(function(event) {
+    if (event.which == 13) {
```

Comment:
Good point. I've changed this to use event.key (which is a string) instead of event.which.